### PR TITLE
Update jquery

### PR DESCRIPTION
**Pull request by ChristianSchmidt1981**
in the code is jquery 1.8.3.

i removed this file. load this with composer.json and update to 1.11.1

I add the file scripts/jquery-browser.js with $.browser

this function is remove in jquery 1.9.0. now it's inside this package

### CODE REVIEW

**Comment by ChristianSchmidt1981** [comment]
without this pull-request jquery >= 1.9.0 not more works

File: index.html [hunk 1]
```diff
@@ -71,7 +71,9 @@
 	<li class="delete separator"><a href="#delete"></a></li>
 </ul>
 
-<script type="text/javascript" src="scripts/jquery-1.8.3.min.js"></script>
+<script type="text/javascript" src="vendor/compoents/jquery/jquery.min.js"></script>
```
Comment (simogeo) [inline index.html]: This is not compliant !

Comment (ChristianSchmidt1981) [inline index.html]: @simogeo 
should i update only the jquery-version and not load from the composer?

Comment (simogeo) [inline index.html]: Yes, the new jQuery version should be in scripts folder and implicitly called from index.html. I know some of you, guys, are waiting for a [package manager version](https://github.com/simogeo/Filemanager/issues/293) (and I understand).

**Comment by ChristianSchmidt1981** [comment]
@simogeo 
load not more from vendor

**Comment by simogeo** [comment]
Thanks, I'll test it locally, then push if ok

**Pull request closed without merging**